app/bundles/CoreBundle/Assets/js/16.columns.js

```diff
@@ -0,0 +1,67 @@
+window.columnConfigs = {
```

Comment:
I'd suggest to have this config in a PHP file so even plugins could hook into this feature. Each bundle can now add JS during the `BUILD_MAUTIC_JS` event so that could be used here as well.